server/src/main/java/io/crate/fdw/ServersMetadata.java

```diff
@@ -158,7 +159,8 @@ public Stream<Option> getOptions() {
 
     private final Map<String, Server> servers;
 
-    ServersMetadata(Map<String, Server> servers) {
+    @VisibleForTesting
+    public ServersMetadata(Map<String, Server> servers) {
```

Comment:
Please don't change visibility but go through the task in the test. Similar to other existing tests